staging/src/k8s.io/kubelet/pkg/apis/podresources/v1/constants.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+const (
+	// Version means current version of the API supported by kubelet
+	Version = "v1"
+	// PodResourcesPath is the folder the Pod Resources is expecting sockets to be on
+	// Only privileged pods have access to this path
+	// Note: Placeholder until we find a "standard path"
+	PodResourcesPath = "/var/lib/kubelet/pod-resources/"
```

Comment:
This constants is problematic because Root Dir `/var/lib/kubelet` could be configured explicitly by user.  see more: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/
> --root-dir string     Default: /var/lib/kubelet
> Directory path for managing kubelet files (volume mounts, etc).

And the subpath has already has an constants here:
https://github.com/kubernetes/kubernetes/blob/65e693eccbc97192c7c96b88b4b446a3ed5f5efc/pkg/kubelet/config/defaults.go#L29

Reply:
Yes, but in deviceplugin api default a like const. In most cases, `Root Dir` is `/var/lib/kubelet`, so this const is still valuable.
https://github.com/kubernetes/kubernetes/blob/f803c112b8e327404e08bca261025cc0c8b7cb2d/staging/src/k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1/constants.go#L30

Reply:
Well, It's make sense to me, and consider that most use case is that running through Pod and read sock file through HostPath.

Reply:
Thank you for your consent 😃

Reply:
I tend to agree with the initial take of @xuzhenglun . The fact the path can change because is constructed from `--root-dir` prevents us to add a meaningful constant.

Even renaming like `default` seems not great to me.

I acknowledge copy pasting the path is tedious, but no alternative seems really better so far (every of them has drawbacks)